### DISCUSSION TITLE
Vue Component Cell Update

### DISCUSSION THREAD

**Discussion by rupertqin** [revolist/revogrid#465] [General]
https://codesandbox.io/s/revogrid-vuecomponent-forked-oz817

when I add more rows to it ,  then I click the buttons to increase the number to "5", scroll down, you see another cell is "5" too.

I think it's a bug of doing duplicate logic here.

**Comment by revolist** [comment]
Hello @rupertqin 
It's not an actual bug.

The thing is there is no object creation for each cell. Initially, Vue templates were introduced only for syntax sugar and it still recommended to use tsx/jsx template instead of vue.

How it works now:
- We do not create vue vnode per each cell. 
- We actually update only properties data and as soon as a cell comes out of viewport it data gets replaced by another cell data but the component by itself still the same. Which provides a super performance.
What you have to do you have to consider editing of row model instead of using data property.

I have provided you with sample in Codebox:
https://codesandbox.io/s/revogrid-vuecomponent-data-hzpqy
